leetcodePython/SlidingWindow/slidingwindow_862.py

```diff
@@ -0,0 +1,29 @@
+from itertools import accumulate
+from typing import List
+from collections import deque
+
+
+class Solution:
+    def shortestSubarray(self, nums: List[int], k: int) -> int:
+        prefix_sum = [0] + list(accumulate(nums))
```

Comment:
Start from here we have an extra `0` at beginning. This makes it easy to do `prefix_sum[i] - prefix_sum[0] = sum to i`